lib/web/fetch/index.js

```diff
@@ -133,116 +176,146 @@ function fetch (input, init = undefined) {
   let requestObject
 
   try {
-    requestObject = new Request(input, init)
-  } catch (e) {
-    p.reject(e)
-    return p.promise
-  }
+    publishIfSubscribed('start', context)
```

Comment:
imho we dont need to put everything into a try catch block as it is unecessary modifying the whitespace.